server/src/components/ConditionalLayout.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
```

Comment:
Don't query file path name. The common header should only be displayed if the user is authenticated. Use the session object here, and convert this to a server component. @thlurte please provide him guidance if necessary.

Reply:
ok,